src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,10 +191,101 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        const regionItm = [];
```

Comment:
Sorry but I dont think this logic would be necessary. Atleast a bulk of it. 
Consider this: 
The region element already has all the data you need (Except axis), for instance in a `multiple line single region scenario` ([here](https://engineering.cerner.com/carbon-graphs/#/regions/line/multiple-line)):

The 2 regions are:

![image](https://user-images.githubusercontent.com/4908886/61481331-abbf3b80-a95d-11e9-8f4d-c139826ea398.png)

When you get the data for these elements:

![image](https://user-images.githubusercontent.com/4908886/61481415-dc9f7080-a95d-11e9-8be5-476a829f121d.png)

The reason why the axis is not available is because the default is Y Axis.

Reply:
This logic is only for paired Result and as we discussed  in the previous PR there were limitation for picking the elements from the dom.